README.md

```diff
@@ -263,6 +282,17 @@ It is likely because the code you are running relies on tensorflow executing sym
 This can be fixed by either changing the order of imports. This is because `import tensorflow as tf` sets eager execution to true, while any `rllib` import disables eager execution. Once the execution state has been set, it cannot be changed. For example, if you require eager execution, make sure `import tensorflow as tf` comes BEFORE `from ray import rllib` and vise versa.
 
 
+## 'human_aware_rl.data_dir' not found
+If you encounter 
+```
+ModuleNotFoundError: No module named 'human_aware_rl.data_dir'
+```
+
+, please run 
 
+```
+./run_tests.sh
+``` 
 
+to initiate those variables
```

Comment:
Honestly now that I think about it, we should probably completely remove all reference to the legacy code in the README as well (basically everything before the "rllib" secion)

Reply:
should be addressed in the next PR. I tried my best